infra/scripts/deploy.sh

```
@@ -65,6 +72,34 @@ wait_readiness() {
+sync_nginx_conf() {
+  local entry name src dst backup
+  local applied=()
+  for entry in "${NGINX_CONF_TARGETS[@]}"; do
+    name="${entry%%:*}"; dst="${entry#*:}"
+    src="$NGINX_SYNC_DIR/$name"
+    [[ -f "$src" ]] || continue   # 서버에서 수동 실행 등 원본이 없으면 건너뜀
+    cmp -s "$src" "$dst" 2>/dev/null && continue
+    backup="$NGINX_SYNC_DIR/$name.prev"
+    [[ -f "$dst" ]] && cat "$dst" > "$backup"
+    log "nginx 설정 갱신: $dst"
+    sudo /usr/bin/tee "$dst" < "$src" >/dev/null
```

Shellcheck (0.11.0)

[info] sudo doesn't affect redirects. Use sudo cat file | ..

(SC2024)